src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
```

Comment:
According to the checklist, avoid using static methods or fields unless they are constants. Here 'BOUND' is a constant which is correctly defined as 'static final'. However, please ensure that 'BOUND' represents a meaningful concept in the context of the 'Lottery' class, such as 'MAX_BALL_NUMBER'.